### PR TITLE
FAB-16958 Remove from README reference to gerrit

### PR DESCRIPTION
**Pull request by TopJohn**
Update readme remove gerrit.
Signed-off-by: TopJohn <xzj19922010@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 [//]: # (SPDX-License-Identifier: CC-BY-4.0)
 
-**Note:** This is a **read-only mirror** of the formal [Gerrit](https://gerrit.hyperledger.org/r/#/admin/projects/fabric-cli) repository,
-where active development is ongoing. Issue tracking is handled in [Jira](https://jira.hyperledger.org/secure/Dashboard.jspa?selectPageId=10104)
+**Note:** Issue tracking is handled in [Jira](https://jira.hyperledger.org/secure/Dashboard.jspa?selectPageId=10104).
 
-This repo is going to be used to implement [FAB-10734 Fabric CLI Redesign](https://jira.hyperledger.org/browse/FAB-10734). This is NOT the "official" Fabric CLI and there is not yet any commitment that it is going to be.
```
Comment (troyronda) [inline README.md]: The removed sentence is still true (so it should not be removed).

File: README.md [hunk 2]
```diff
@@ -12,7 +11,7 @@ The Hyperledger Fabric CLI is a tool used to interact with [Fabric networks](htt
 ## Installation
 
 1. Clone this repo
-2. Install `gobin` using `GO111MODULE=off go get -u github.com/myitcv/gobin`
+2. Install `gobin` using `GO111MODULE=off go Update README with CI badge and remove gerritget -u github.com/myitcv/gobin`
```
Comment (troyronda) [inline README.md]: It looks like some extra text got copied in?

Comment (TopJohn) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (lehors) [reply README.md]: Agreed!

**Pull request merged**